### ISSUE TITLE
[idea] MD headings as note title and for (back)linking

### ISSUE THREAD

**Issue by soelderer** [elias-sundqvist/obsidian-annotator#256]
Hi, let me first thank you for the great work!

I have a feature request to introduce MD headings as note titles.
This would have several benefits:
1. Titles help to keep an overview of longer notes
2. Allows for human readable and fully MD compatible links to specific notes (e.g. for use in MOCs)
3. Allows for backlinks
4. Makes the MD files better readable when opened in MD mode (e.g. Outline, collapse all headings)
5. Allows to structure the note file hierarchically (e.g. in scientific papers: "Methods" - "Sampling" - "My specific note on sampling in the study" / "my other note on sampling" / ...)
6. Makes it easier to use the MD note in other plugins like [mind map](https://github.com/lynchjames/obsidian-mind-map)

The feature request is related to and fits nicely with ideas like #32 #24 #76.

Especially for people doing scholarly work annotating a lot of papers (and re-annotating in the future) this would be a big help, especially the human readable links for MOCs. E.g. it allows to reference the results of many annotated studies to create an overview MD file. Also in many cases, reviewing the annotations or headings without opening the PDFs is enough.

The request is inspired by [Org-roam-bibtex](https://github.com/org-roam/org-roam-bibtex). See e.g. this [video](https://www.youtube.com/watch?v=Wy9WvF5gWYg)  (approx. at 4:11).

I don't know anything about the backend and if the request is doable, so it's just an idea.

All the best!
Paul

**Comment by JackJackAgain** [comment]
I agree, this would be a nice feature that would facilitate everyday use and scholarly annotation.
Hope this is doable

**Comment by soelderer** [comment]
I just discovered that it's actually possible to introduce headings manually in MD mode given that you put empty lines before and after the headings. This allows for all the benefits I mentioned above.

In combination with the [Force note view mode plugin](https://github.com/bwydoogh/obsidian-force-view-mode-of-note) to force preview mode it's actually pretty neat for an academic workflow.

Still, official implementation would be nice to have.
